server/spec/spec_helper.rb

```diff
@@ -1,6 +1,3 @@
-require "simplecov"
-SimpleCov.start
-
```

Comment:
These two lines are required for the coverage calculation and need to be put back in.